importer/NlGemNl.py

```diff
@@ -0,0 +1,157 @@
+                for wl in wikilinks:
+                    arch_q = utils.q_from_wikipedia("nl", wl.title)
+                    if arch_q:
+                        self.add_statement("architect", arch_q)
+
+    def set_inception(self):
+        """
+        Set building date.
+
+        Only if it's a single, full year, i.e. no "ca." or ranges.
+        """
+        if (self.has_non_empty_attribute("bouwjaar") and
+           utils.legit_year(self.bouwjaar)):
+            self.add_statement(
+                "inception", {"time_value": {"year": self.bouwjaar}})
+
+    def set_address(self):
+        """
+        Set Located at street address.
+
+        Validation: must contain digit.
+        Many instances of addresses within {{sorteer}} template,
+        so we strip those first.
+        """
+        street_address = False
+        if self.has_non_empty_attribute("adres"):
+            if len(utils.wparser.parse(self.adres).filter_templates()) == 1:
+                address_try = utils.wparser.parse(
+                    self.adres).filter_templates()[0].params[0]
+                if utils.contains_digit(address_try):
+                    street_address = address_try
+            else:
+                if utils.contains_digit(self.adres):
+                    street_address = self.adres
+        if street_address:
+            self.add_statement("located_street", street_address)
+
+    def set_heritage_id(self):
+        """
+        Set the WLM ID.
+
+        The ID's used in the lists are made up, since
+        there's no general register of municipal monuments
+        (each municipality does their own).
+        """
+        country = self.mapping["country_code"].upper()
+        wlm_id = "{}-{}".format(country, self.monuments_all_id)
+        self.add_statement("wlm_id", wlm_id)
+
+    def update_labels(self):
+        """
+        Set labels in Dutch.
+
+        In most cases, we use the (de-markuped if needed)
+        object field. There's a fair number of momuments
+        without this field filled out, but those always
+        have the street address, so that one is used
+        as a label instead.
+        """
+        if self.object == "":
+            label_material = self.adres
+        else:
+            label_material = self.object
+        self.add_label("nl",
+                       utils.remove_markup(label_material))
+
+    def update_descriptions(self):
+        """
+        Set descriptions in several languages.
+
+        Attempt to use correct placenames
+        in Dutch/Frisian.
+        """
+        desc_dict = {
+            "nl": "gemeentelijk monument in {}",
+            "fy": "gemeentlik monumint yn {}",
+            "en": "municipal monument in {}"
+        }
+        placenames = {
+            "en": "the Netherlands",
+            "nl": "Nederland",
+            "fy": "Nederlân"
+        }
+        for lang in desc_dict:
+            try:
+                placename = [x[lang] for x
```

Comment:
You don't append the country name to the description when a municipality name is found?

Reply:
No, that would be too long.

Reply:
ok